docs/event-types/overview.mdx

```diff
@@ -21,3 +23,18 @@ description: "Overview of all Flare API event types"
 | `seller`        | [Seller <Icon icon="book" size={16} />](/event-types/seller)                    |
 | `source_code`   | [Source code <Icon icon="book" size={16} />](/event-types/source-code)          |
 | `stealer_log`   | [Stealer Log <Icon icon="book" size={16} />](/event-types/stealer-log)          |
+
+# Other Event Types
```

Comment:
Why are those not primary?

As a user, what am I supposed to understand here?


If you think they really should be in another table, then we could add a small sentence that explains why this table is different